src/events/render-advert-label.ts

```diff
@@ -7,15 +7,21 @@ import { getCookie } from '@guardian/libs';
 import fastdom from 'utils/fastdom-promise';
 import crossIcon from '../../static/svg/icon/cross.svg';
 
-const shouldRenderLabel = (adSlotNode: HTMLElement): boolean => {
-	if (
-		adSlotNode.classList.contains('ad-slot--fluid') &&
-		!adSlotNode.classList.contains('ad-slot--interscroller') &&
-		!adSlotNode.classList.contains('ad-slot--article-end')
-	) {
+const templatesWithoutLabels = [
+	10077207, // CAPI_MULTIPLE_HOSTED
+	10069167, // CAPI_MULTIPLE_PAIDFOR
+	12317037, // EVENTS_MULTIPLE
+	10070487, // CAPI_SINGLE_PAIDFOR
+	10063287, // MANUAL_MULTIPLE
+];
+
+const shouldRenderLabel = (
+	adSlotNode: HTMLElement,
+	creativeTemplateId: number,
```

Comment:
I think this should be optional because most adverts don't even have a `creativeTemplateId`, coercing it to `0` below doesn't feel right.

Reply:
I was thinking the same when I was updating the tests with new paramater.